terraform/cloudfoundry/network_acls_cf_in.tf

```diff
@@ -172,6 +172,17 @@ resource "aws_network_acl_rule" "117_kibana" {
     egress = false
 }
 
+resource "aws_network_acl_rule" "118_graphite" {
+    network_acl_id = "${aws_network_acl.cf.id}"
+    protocol = "tcp"
+    rule_number = 118
```

Comment:
It's going to be rule clash: https://github.com/alphagov/paas-cf/blob/113996375_ha_for_logsearch/terraform/cloudfoundry/network_acls_cf_in.tf#L178